common/common.go

```diff
@@ -29,6 +29,8 @@ func GetNetwork(network string) (string, error) {
 		return MainNetwork, nil
 	case strings.ToLower(TestNetwork), strings.ToLower("TestNet"):
 		return TestNetwork, nil
+	case strings.ToLower(UnitTestNetwork), strings.ToLower("UnitTest"):
```

Comment:
is there a reason to make a new network for this?

Reply:
Yea, the unit tests were using "TestNet" before. This means the Testnet activation heights were being used, which was mainly v1 for all unit tests. I removed some of the v1 stuff from the polling, since we don't need to query XPD and XPT prices anymore. This broke the unit tests that expected the v1 asset lists. The `UnitTest` network has a helper function that takes w/e version I set it to at the start of the unit test. So now when I run unit tests, I use this network, and run a function to decide what version is "active"